### PR TITLE
Add community health documents

### PR DESCRIPTION
**Pull request by webknjaz**

### CODE REVIEW

**Review by cidrblock — APPROVED** [review]
approved

I would like to see this in, and appreciate the intent of the contribution checklist, but am concerned it might come across demeaning to some.  Eager to see other feedback here

**Comment by relrod** [comment]
>  appreciate the intent of the contribution checklist, but am concerned it might come across demeaning to some

I made a similar concern to @webknjaz internally. I think the PR template should be as simple/minimal as possible, and I think the checklist risks putting off new contributors. It just reads as though there are so many hoops to jump through to submit a simple patch. I would do away with it.

**Review by relrod — CHANGES_REQUESTED** [review]
I'm actually going to block this now that I know I'm not the only one concerned about the checklist.

**Comment by webknjaz** [comment]
@relrod @cidrblock would you help me come up with a better-worded checklist? It's mostly a compilation of the templates from other projects I maintain or contribute to. Is there any way to make it sound better? Maybe an explicit statement that only the linted things are important? I like having the checklist because it states some expectations and provides advice. I recognize that it's not ideal but struggle to make it better on my own. I also don't want to send people to external documents too much.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+3. Send it to us as a PR.
+
+4. Iterate on your PR, incorporating the requested improvements
+   and participating in the discussions.
+
+Prerequisites:
+
+1. Have [npm].
+
+2. Use [npm] run the tests.
```
[the remark this answers is not in the document]
Comment (webknjaz) [reply .github/CONTRIBUTING.md:19]: This was meant to be a minimal skeleton for many pages, partially copied from other projects. I'd love for them to be improved in separate PRs. I just want to iterate on the improvements quickly.

**Comment by webknjaz** [comment]
Note that if a lot of those requests are not fulfilled, we'd have to ask to improve the quality of the contribution and in extreme cases even to understand what's going on which may be perceived as nitpicking by the contributors but if they see the expectations, it's easier on them because they'd know what to do w/o a human telling them to do that.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<!-- markdownlint-disable first-line-heading -->
+<!-- Thank you for your contribution! -->
+<!-- First time contributors: Take a moment to review https://als.rtfd.io/en/latest/contributing/guidelines! -->
+
+❓ **What kind of change does this PR introduce?**
+
+* [ ] 🐞 bug fix
```
Comment (ssbarnea) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: What is the real benefit of having this question?

Comment (webknjaz) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: It's nice to see the intent/perception of the submitter. There's still no forms for PRs.

**Pull request merged**